bot/messages.ts

```
@@ -355,33 +366,41 @@ const beginTakeBuyMessage = async (ctx: MainContext, bot: HasTelegram, seller: U
-  randomImage: IOrder["random_image"]
+  randomImage: string,
+  isGoldenHoneyBadger: boolean = false
 ) => {
   try {
-    const currencyObject = getCurrency(fiatCode);
-    const currency =
-      !!currencyObject && !!currencyObject.symbol_native
-        ? currencyObject.symbol_native
+    const currency = getCurrency(fiatCode);
+    const currencySymbol = 
+      (currency !== null && currency.symbol_native)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)